packages/headless/src/controllers/facets/facet/headless-facet.ts

```diff
@@ -165,7 +173,7 @@ export function buildFacet(
       dispatch(executeSearch(logFacetShowLess(facetId)));
     },
 
-    /** @returns The state of the `Facet` controller. */
+    /** @returns (FacetState) The state of the `Facet` controller. */
     get state() {
```

Comment:
In the return statement of this getter, all the properties are annotated with `@returns`, which I wouldn't expect because they're properties. Just want to check that this is the correct syntax and not a mistake before I apply this to other controllers.

Reply:
I think you are right. My first instinct is `@returns` should be used only for functions, not properties or even getters (since they are derived properties). I would check jsdoc reference examples online to make sure.

"Looking at documentation on how to write documentation" 😄